### ISSUE TITLE
Qbittorrent settings dont save on restart

### ISSUE THREAD

**Issue by mgherghi** [MichaIng/DietPi#1957]
# Solution: https://github.com/Fourdee/DietPi/issues/1957#issuecomment-407173961

for some reason i cant get the settings on the app to stick after a reboot.

**Comment by Fourdee** [comment]
@mgherghi 

Thanks for the report 👍 

We'll open for investigations.

**Comment by Fourdee** [comment]
Confirmed issue, we'll look for a solution.

**Comment by Fourdee** [comment]
## Solution:
```
chown -R qbittorrent:dietpi /home/qbittorrent
dietpi-services restart
```

**Comment by Fourdee** [comment]
@mgherghi 

We'll get this resolved for v6.13, however, you can fix it now by using the following commands:
https://github.com/Fourdee/DietPi/issues/1957#issuecomment-407173961

**Comment by Fourdee** [comment]
Completed, patched in v6.13.
